droid-command-line/src/main/java/uk/gov/nationalarchives/droid/command/ResultPrinter.java

```diff
@@ -1,273 +1,329 @@
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package uk.gov.nationalarchives.droid.command;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+import uk.gov.nationalarchives.droid.command.action.CommandExecutionException;
+import uk.gov.nationalarchives.droid.command.archive.ArchiveConfiguration;
+import uk.gov.nationalarchives.droid.command.archive.Bzip2ArchiveContentIdentifier;
+import uk.gov.nationalarchives.droid.command.archive.GZipArchiveContentIdentifier;
+import uk.gov.nationalarchives.droid.command.archive.IsoArchiveContainerIdentifier;
+import uk.gov.nationalarchives.droid.command.archive.RarArchiveContainerIdentifier;
+import uk.gov.nationalarchives.droid.command.archive.SevenZipArchiveContainerIdentifier;
+import uk.gov.nationalarchives.droid.command.archive.ZipArchiveContentIdentifier;
+import uk.gov.nationalarchives.droid.command.archive.TarArchiveContentIdentifier;
+import uk.gov.nationalarchives.droid.command.archive.ArcArchiveContentIdentifier;
+import uk.gov.nationalarchives.droid.command.archive.WarcArchiveContentIdentifier;
+import uk.gov.nationalarchives.droid.command.container.Ole2ContainerContentIdentifier;
+import uk.gov.nationalarchives.droid.command.container.ZipContainerContentIdentifier;
+import uk.gov.nationalarchives.droid.container.ContainerFileIdentificationRequestFactory;
+import uk.gov.nationalarchives.droid.container.ContainerSignatureDefinitions;
+import uk.gov.nationalarchives.droid.container.TriggerPuid;
+import uk.gov.nationalarchives.droid.container.ole2.Ole2IdentifierEngine;
+import uk.gov.nationalarchives.droid.container.zip.ZipIdentifierEngine;
+import uk.gov.nationalarchives.droid.core.BinarySignatureIdentifier;
+import uk.gov.nationalarchives.droid.core.interfaces.IdentificationRequest;
+import uk.gov.nationalarchives.droid.core.interfaces.IdentificationResult;
+import uk.gov.nationalarchives.droid.core.interfaces.IdentificationResultCollection;
+import uk.gov.nationalarchives.droid.core.interfaces.archive.IdentificationRequestFactory;
+
+/**
+ * File identification results printer.
+ *
+ * NB: This class is called recursively when archive files are opened 
+ * 
+ * @author rbrennan
+ */
+public class ResultPrinter {
+
+    private static final String R_SLASH = "/";
+    private static final String DOUBLE_SLASH = "\\";
+    private static final String L_BRACKET = "(";
+    private static final String R_BRACKET = ")";
+    private static final String SPACE = " ";
+
+    private static final String OLE2_CONTAINER = "OLE2";
+    private static final String ZIP = "ZIP";
+    private static final String ZIP_ARCHIVE = "x-fmt/263";
+    private static final String RAR_ARCHIVE = "x-fmt/264";
+    private static final String OTHERRAR_ARCHIVE = "fmt/411";
+    private static final String TAR_ARCHIVE = "x-fmt/265";
+    private static final String GZIP_ARCHIVE = "x-fmt/266";
+    private static final String ARC_ARCHIVE = "x-fmt/219";
```

Comment:
BZip x-fmt/267 is missing, should that be included as well?

Reply:
yes it should. fixed in https://github.com/digital-preservation/droid/pull/356/commits/e3f376bb3eaa2d44d4ff9a397952b7a5f81d47fe